entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md

```diff
@@ -19,6 +19,316 @@ EF7 is also available as [daily builds](https://github.com/dotnet/efcore/blob/ma
+
+However, keep in mind that:
+
+- The specific changes to make must be specified explicitly; they are not automatically detected by EF Core.
+- Any tracked entities will not be kept in sync.
+- Additional commands may need to be sent in the correct order so as not to violate database constraints. For example deleting dependents before a principal can be deleted.
+
+All of this means that the `ExecuteUpdate` and `ExecuteDelete` methods compliment, rather than replace, the existing `SaveChanges` mechanism.
+
+### Sample model
+
+The examples below use a simple model with blogs, posts, tags, and authors:
+
+<!--
+public class Blog
+{
+    public Blog(string name)
+    {
+        Name = name;
+    }
+
+    public int Id { get; private set; }
+    public string Name { get; set; }
+    public List<Post> Posts { get; } = new();
+}
+
+public class Post
+{
+    public Post(string title, string content, DateTime publishedOn)
+    {
+        Title = title;
+        Content = content;
+        PublishedOn = publishedOn;
+    }
+
+    public int Id { get; private set; }
+    public string Title { get; set; }
+    public string Content { get; set; }
+    public DateTime PublishedOn { get; set; }
+    public Blog Blog { get; set; } = null!;
+    public List<Tag> Tags { get; } = new();
+    public Author? Author { get; set; }
+}
+
+public class FeaturedPost : Post
+{
+    public FeaturedPost(string title, string content, DateTime publishedOn, string promoText)
+        : base(title, content, publishedOn)
+    {
+        PromoText = promoText;
+    }
+
+    public string PromoText { get; set; }
+}
+
+public class Tag
+{
+    public Tag(string text)
+    {
+        Text = text;
+    }
+
+    public int Id { get; private set; }
+    public string Text { get; set; }
+    public List<Post> Posts { get; } = new();
+}
+
+public class Author
+{
+    public Author(string name)
+    {
+        Name = name;
+    }
+
+    public int Id { get; private set; }
+    public string Name { get; set; }
+    public List<Post> Posts { get; } = new();
+}
+-->
+[!code-csharp[BlogsModel](../../../../samples/core/Miscellaneous/NewInEFCore7/BlogsContext.cs?name=BlogsModel)]
+
+> [!TIP]
+> The sample model can be found in [BlogsContext.cs](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Miscellaneous/NewInEFCore7/BlogsContext.cs).
+
+### Basic `ExecuteDelete` examples
+
+> [!TIP]
+> The code shown here comes from [ExecuteDeleteSample.cs](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Miscellaneous/NewInEFCore7/ExecuteDeleteSample.cs).
+
+Calling `ExecuteDelete` or `ExecuteDeleteAsync` on a `DbSet` will immediately delete all entities of that `DbSet` from the database. For example, to delete all `Tag` entities:
```

Comment:
Generally try to avoid the future tense

```suggestion
Calling `ExecuteDelete` or `ExecuteDeleteAsync` on a `DbSet` immediately deletes all entities of that `DbSet` from the database. For example, to delete all `Tag` entities:
```